Source/Moya+Defaults.swift

```diff
@@ -3,8 +3,8 @@ import Alamofire
 /// These functions are default mappings to `MoyaProvider`'s properties: endpoints, requests, manager, etc.
 public extension MoyaProvider {
     public final class func DefaultEndpointMapping(target: Target) -> Endpoint<Target> {
-        let url = target.baseURL.URLByAppendingPathComponent(target.path).absoluteString
-        return Endpoint(URL: url, sampleResponseClosure: {.NetworkResponse(200, target.sampleData)}, method: target.method, parameters: target.parameters)
+        let url = target.baseURL.URLByAppendingPathComponent(target.path)!.absoluteString
```

Comment:
Hmm, what if we replaced this with a check for a `nil` URL? Something like a `guard` that `else`es to a `fatalError`, then we can avoid the force unwraps?